lib/pacemaker/pcmk_sched_bundle.c

```diff
@@ -283,10 +284,12 @@ pcmk__bundle_internal_constraints(pe_resource_t *rsc,
             replica->ip->cmds->internal_constraints(replica->ip, data_set);
 
             // Start IP then container
-            pcmk__order_resource_actions(replica->ip, RSC_START, replica->container, RSC_START,
-                          pe_order_runnable_left|pe_order_preserve, data_set);
-            pcmk__order_resource_actions(replica->container, RSC_STOP, replica->ip, RSC_STOP,
-                          pe_order_implies_first|pe_order_preserve, data_set);
+            pcmk__order_starts(replica->ip, replica->container,
+                               pe_order_runnable_left|pe_order_preserve,
+                               data_set);
+            pcmk__order_stops(replica->container, replica->ip,
+                              pe_order_implies_first|pe_order_preserve,
+                              data_set);
```

Comment:
These are the only two that switched from pcmk__order_resource_actions to pcmk__order_*.  Was that intentional?

Reply:
Yep, the latter are convenience macros wrapping the former, and this was a place they could be used but weren't